bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-reply-intention/component.tsx

```
@@ -85,13 +85,10 @@ const ChatReplyIntention = () => {
     >
       <Styled.Message>
-        <Styled.Markdown
-          linkTarget="_blank"
-          allowedElements={window.meetingClientSettings.public.chat.allowedElements}
-          unwrapDisallowed
-        >
-          {messageToQuoteMarkdown(message)}
-        </Styled.Markdown>
+        <Styled.HtmlContent
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={{ __html: getFirstVisibleLineHtml(message || '') }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)